content/SCALE/DataProtection/Replication.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Replication"
+weight: 100
+---
+
+## Remote Replication
+
+
+{{< toc >}}
+
+Configure SSH in TrueNAS before creating a remote replication task. This ensures that both systems can connect to each other and new snapshots are regularly available for replication.
+
+To streamline creating simple replication configurations, the replication wizard assists with creating a new SSH connection and automatically creates a periodic snapshot task for sources that have no existing snapshots.
+
+{{< expand "Process Summary" "v" >}}
+## Process Summary
+
+* **Tasks > Replication Tasks**
+  * Choose sources for snapshot replication.
+    * Remote sources require an SSH connection.
+    * TrueNAS shows how many snapshots will be replicated.
+  * Define the snapshot destination.
+    * A remote destination requires an SSH connection.
+    * Choose destination or define manually by typing a path.
+      * Adding a new name on the end of the path creates a new dataset.
+  * Choose replication security.
+    * We always recommend Replication with encryption.
+    * Disabling encryption is only meant for absolutely secure networks.
```

Comment:
If we are talking about zfs dataset encryption then it's probably "Disabling encryption is only meant for absolutely secure and trusted destinations."